lib/cell.rb

```diff
@@ -0,0 +1,32 @@
+class Cell
+
+  attr_accessor :value, :neighbours, :solutions
+
+  CELL_VALUES = [1,2,3,4,5,6,7,8,9]
+
+  def initialize(value)
+    raise 'Values must be less than 10!' unless value.length == 1
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.